airflow_providers_wherobots/operators/run.py

```diff
@@ -143,8 +145,16 @@ def execute(self, context) -> Any:
         Trigger the Wherobots Run and keep polling for status until the Run ends
         """
         with WherobotsRestAPIHook(self.wherobots_conn_id) as rest_api_hook:
-            self.log.info(f"Creating Run with payload {self.run_payload}")
-            run = rest_api_hook.create_run(self.run_payload)
+            if not self.region:
```

Comment:
Could we move this to a common helper function to be able to change messaging and default in a single place? (even though this will be deprecated soon ™️ )